src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,43 @@
+package racingcar.domain
+
+import racingcar.domain.Location.Companion.initialLocation
+
+typealias CarId = Long
+
+interface CarRef {
+    val carId: CarId
+}
+
+class Car(override val carId: CarId) : CarRef {
+    private var location: Location = initialLocation
+
+    fun location(): Location = location
```

Comment:
객체의 상태를 나타낸다면 [함수 대신 프로퍼티](https://kotlinlang.org/docs/coding-conventions.html#functions-vs-properties)로 제공하면 어떨까요?

```suggestion
    var location: Location = initialLocation
        private set
```